docs/bookstore/page-0.5/pg0.5-env-setup.md

```diff
@@ -0,0 +1,229 @@
+
+Run the following command and keep the terminal open:
+
+```shell
+minikube tunnel
+```
+
+???+ success "Verify"
+    If there aren't any error messages, it means you have set up the tunnel successfully.
+
+### **Task 2: Running the Bookstore Web App**
+
+![Image](images/image12.png)
+
+The Next.js frontend app is located in the `docs/code-samples/eventing/bookstore-sample-app/start/frontend` folder.
+
+Ensure that port 3000 on your local machine is not being used by another application.
+
+#### **Deploy the Frontend App**
+
+You can either [build the image locally](https://docs.docker.com/get-started/02_our_app/) or use our pre-built image. If you are using the pre-built image, you can proceed to the next step.
+
+When ready, run the following command to deploy the frontend app:
+
+```shell
+kubectl apply -f config/100-front-end-deployment.yaml
+```
+
+![Image](images/image11.png)
+
+This command will pull the image and deploy it to your cluster as a Deployment. It will also expose it as a LoadBalancer to receive external traffic.
+
+```shell
+kubectl get pods
+```
+
+You will see that your front end pod is running.
+
+![Image](images/image4.png)
+
+#### **Port Forwarding (Optional under condition)**
+
+![Image](images/image9.png)
+
+You might need to set up port forwarding to access the app from your local machine.
+
+Check if port forwarding is necessary by running:
+
+```shell
+kubectl get services -A
+```
+
+![Image](images/image1.png)
+
+If the `EXTERNAL_IP` for your frontend service is `127.0.0.1`, port forwarding is not needed.
+
+If port forwarding is required, open a new terminal and run:
+
+```shell
+kubectl port-forward svc/bookstore-frontend-svc 3000:3000
+```
+
+![Image](images/image8.png)
+
+Don’t close the terminal when port-forwarding is established.
+
+???+ success "Verify"
+
+    Visit [http://localhost:3000](http://localhost:3000) in your browser. The UI page should appear!
+
+    ![Image](images/image19.png)
+
+
+
+
+
+### **Task 3: Running the Book Review Service**
+
+![Image](images/image6.png)
+
+The Node.js server is located in the `docs/code-samples/eventing/bookstore-sample-app/start/node-server` folder.
+
+Ensure that port 8080 on your local machine is not being used by another application.
+
+#### **Deploy the Book Review Service: Node.js Server**
+
+You can either [build the image locally](https://docs.docker.com/get-started/02_our_app/) or use our pre-built image. If you are using the pre-built image, you can proceed to the next step.
+
+When ready, run the following command to deploy the Node.js server:
+
+```shell
+kubectl apply -f config/100-deployment.yaml
+```
+
+This command will pull the image and deploy it to your cluster as a Deployment. It will also expose it as a LoadBalancer to receive external traffic.
+
+![Image](images/image15.png)
+
+```shell
+kubectl get pods
+```
+
+You will see that your node-server pod is running.
```

Comment:
```suggestion
You will see that your Node.js server pod is running.
```

Reply:
Updated to Node.js server (node-server), as the deployment and pod name should be `node-server-xxxx`